### ISSUE TITLE
Toggle between Raw / Calibrated value display

### ISSUE THREAD

**Issue by amitschang** [ssec-jhu/evolver-ng#209]
On the state page. Default should probably be to show non-raw. I don't think the raw is a well-defined concept, but at least we can take convention. Comes from wishlist

**Comment by imaitland** [comment]
yeah this is currently like this because since there's no calibration procedure and fits there's no calibrated data to display. Once calibration procedure's have been implemented we can start showing the non-raw values on the state display.
